tools/cache-build.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+# Travis packs all cache directories into one archive for caching.
+# So, we put files into tar, not tar.gz to improve compression
+# (because the files are shared between mongooseim and big_tests)
+
+set -e
+CACHE_DIR=~/.cache/mim_builds
+
+## Use array assignment to get just hash without filename
+## https://stackoverflow.com/a/5773761
+SHA_ARRAY=($(cat "rebar.config" "rebar.lock" | shasum))
```

Comment:
what about `big_tets/rebar.config` ?

Reply:
There is tools/cache-tests.sh for caching big_tests/_build.

Reply:
sorry, you're right, I didnt see it at this point of review :D